ci/pipelines/deploy-to-test.yml

```diff
@@ -1919,6 +1952,18 @@ jobs:
         params:
           image: image/image.tar
           additional_tags: nginx-forward-proxy-git-release/.git/ref
+  - name: push-nginx-forward-proxy-to-staging-ecr
+    plan:
+      - get: pay-ci
+      - get: nginx-forward-proxy-ecr-registry-test
+        params:
+          format: oci
+        trigger: true
+        passed: [deploy-frontend]
```

Comment:
this could actually be smoke test frontend, since we do have smoke tests for frontend

Reply:
Good shout yes will change that.